apps/fulltextsearch.sh

```diff
@@ -6,6 +6,7 @@
 # shellcheck disable=2034,2059
 true
 SCRIPT_NAME="Full Text Search"
+export SCRIPT_EXPLAINER="Full Text Search provides Elasticsearch for Nextcloud, which makes it possible to search for text inside files."
```

Comment:
You don't need to export it

Reply:
let's do it like this, then we can test.

Less code, better function. :partying_face: